spring-cloud-azure-samples/feature-management-web-sample/src/main/java/com/example/TargetingContextAccessor.java

```diff
@@ -5,29 +5,33 @@
  */
 package com.example;
 
-import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
 
 import javax.servlet.http.HttpServletRequest;
 
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.util.StringUtils;
 
 import com.microsoft.azure.spring.cloud.feature.manager.targeting.ITargetingContextAccessor;
 import com.microsoft.azure.spring.cloud.feature.manager.targeting.TargetingContext;
 
 import reactor.core.publisher.Mono;
 
 public class TargetingContextAccessor implements ITargetingContextAccessor {
-    
+
     @Autowired
     private HttpServletRequest requestContext;
 
     @Override
     public Mono<TargetingContext> getContextAsync() {
         TargetingContext context = new TargetingContext();
         context.setUserId(requestContext.getParameter("User"));
-        ArrayList<String> groups = new ArrayList<String>();
-        groups.add("Ring3");
-        context.setGroups(groups);
+        String group = requestContext.getParameter("Group");
```

Comment:
Looks like we updated to support multiple groups. If that's the case the parameter should be `Groups`

/foo?User=Jeff&Group=Ring0,BetaUsers